docs/concepts/storage/persistent-volumes.md

```diff
@@ -94,6 +95,28 @@ Finalizers:    [kubernetes.io/pvc-protection]
 ...
 ```
 
+You can see that a PV is protected when the PV's status is `Terminating` and the `Finalizers` list includes `kubernetes.io/pv-protection` too:
+
+```shell
+$ kubectl describe pv task-pv-volume
```

Comment:
please remove the `$` - per https://kubernetes.io/docs/home/contribute/style-guide/#dont-include-the-command-prompt

Reply:
done